graphql/appSettings.graphql

```diff
@@ -1,5 +1,5 @@
 query AppSettings {
-  appSettings {
+  appSettings @context(provider: "vtex.reviews-and-ratings") {
```

Comment:
The CI had a bug that reported a missing @context here, but it was a bug in it, it shouldn't have.

@kaisermann worked on a fix in the CI, and now it's fine.

I'm not sure if keeping this @context here works, did you test it? You can remove it if you want though

Reply:
Just tested and everything seems to be working fine. Perhaps best I remove it since it is unnecessary here.

Reply:
@klzns This query is imported by the React component, so I think it's appropriate to add the context. The bug was that the CI was reporting the missing @context for the schema as well.

Reply:
Oh, we should move this query to the react folder then. We should only keep the schema in this folder.

Reply:
Good point. There was another app that used this pattern that we had used as a basis, but I can't remember which one it was. Moved.